docs/pages/includes/database-access/db-configure-start.mdx

```diff
@@ -0,0 +1,127 @@
+{{ dbName="test" }}
+<Tabs>
+<TabItem label="Using a config file">
+On the host where you will run the Teleport Database Service, start Teleport
+with the appropriate configuration.
+
+Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
+
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=teleport.example.com:443 \
+   --name={{ dbName }} \
+   --protocol={{ dbProtocol }} \
+   --uri={{ databaseAddress }} \
+   --labels=env=dev 
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=mytenant.teleport.sh:443 \
+   --name={{ dbName }} \
+   --protocol={{ dbProtocol }} \
+   --uri={{ databaseAddress }} \
+   --labels=env=dev
+```
+
+</ScopedBlock>
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  A single Teleport process can run multiple different services, for example
+  multiple Database Service agents as well as the SSH Service or Application
+  Service.
+
+</Admonition>
+
+Configure the Database Service to start automatically when the host boots up by
+creating a systemd service for it. The instructions depend on how you installed
+the Database Service.
+
```

Comment:
I think we have an existing partial file for this that we could embed here. `docs/pages/includes/start-teleport.mdx`

Reply:
You can't put include files in includes or would have.  It just outputs the text of the include directive.